code/drasil-gool/lib/Drasil/GOOL/LanguageRenderer/CLike.hs

```diff
@@ -88,6 +94,8 @@ andOp = andPrec "&&"
 orOp :: (Monad r) => VSOp r
 orOp = orPrec "||"
 
+inOp :: (Monad r) => VSOp r
+inOp = inPrec ""
```

Comment:
A comment for why the empty string here is the right thing would be good.

Reply:
This was originated used in python but this has been deleted